### ISSUE TITLE
--debug builds crash when executing the unit tests

### ISSUE THREAD

**Issue by mmomtchev** [contra/node-gdal-next#26]
The --debug builds, at least with g++ 8, crash with a segfault while doing the unit tests

This test is the culprit:

```js
const gdal = require('../node-gdal-next')

const src = gdal.open(`sample.tif`)

const options = {
    src: src,
    s_srs: src.srs,
    t_srs: gdal.SpatialReference.fromEPSG(4326),
    cutline: new gdal.LineString()
}
const info = gdal.suggestedWarpOutput(options)

options.dst = gdal.open(
    'temp',
    'w',
    'MEM',
    info.rasterSize.x,
    info.rasterSize.y,
    1,
    gdal.GDT_Byte
)
options.dst.geoTransform = info.geoTransform

gdal.reprojectImage(options)
```

**Comment by mmomtchev** [comment]
osgeo/gdal#3037

**Comment by mmomtchev** [comment]
Confirmed fixed after GDAL upgrade to 3.2.0
